packages/core/src/internal/deployment-loader/file-deployment-loader.ts

```diff
@@ -117,6 +117,26 @@ export class FileDeploymentLoader implements DeploymentLoader {
     );
   }
 
+  public async readBuildInfo(futureId: string): Promise<BuildInfo> {
```

Comment:
This is added onto the file deployment loader but the deployment loader interface right? So we aren't expanding the api, just leveraging the file deployment loader as the most sensible place to put the `readBuildInfo` logic?

Reply:
Correct. This did require disabling the type test for `FileDeploymentLoader`, however.